partiql-tests-data/eval/query/order-by.ion

```diff
@@ -232,6 +232,122 @@ simple::[
       output:[{'supplierId': 10, n: 3}, {'supplierId': 11, 'n':2}]
     }
   },
+  // should order by aggregation function directly
+  {
+    name:"order by aggregation function",
+    statement:"SELECT supplierId, MAX(price) FROM products GROUP BY supplierId ORDER BY MAX(price) DESC",
+    assert:{
+      evalMode:[EvalModeCoerce, EvalModeError],
+      result:EvaluationSuccess,
+      output:[{'supplierId': 10, 'MAX': 15.0}, {'supplierId': 11, 'MAX': 15.0}]
+    }
+  },
+]
+
+alias::[
+  // ORDER BY alias from SELECT
+  {
+    name:"order by select alias",
+    statement:"SELECT productId AS pid, price AS cost FROM products ORDER BY pid",
+    assert:{
+      evalMode:[EvalModeCoerce, EvalModeError],
+      result:EvaluationSuccess,
+      output:[
+        {'pid': 1, 'cost': 5.0},
+        {'pid': 2, 'cost': 10.0},
+        {'pid': 3, 'cost': 15.0},
+        {'pid': 4, 'cost': 5.0},
+        {'pid': 5, 'cost': 15.0}
+      ]
+    }
+  },
+  // ORDER BY multiple aliases
+  {
+    name:"order by multiple aliases",
+    statement:"SELECT supplierId AS sid, price AS cost FROM products ORDER BY cost DESC, sid ASC",
+    assert:{
+      evalMode:[EvalModeCoerce, EvalModeError],
+      result:EvaluationSuccess,
+      output:[
+        {'sid': 10, 'cost': 15.0},
+        {'sid': 11, 'cost': 15.0},
+        {'sid': 10, 'cost': 10.0},
+        {'sid': 10, 'cost': 5.0},
+        {'sid': 11, 'cost': 5.0}
+      ]
+    }
+  },
+  // ORDER BY aggregation alias
+  {
+    name:"order by aggregation alias",
+    statement:"SELECT supplierId, MAX(price) AS max_price FROM products GROUP BY supplierId ORDER BY max_price DESC",
```

Comment:
Since the `max_price` has duplicate values, should also specify an ordering for `supplierId`.